atc/atccmd/command.go

```diff
@@ -279,7 +283,7 @@ func (m *Migration) Execute(args []string) error {
 	if m.OldEncryptionKey.AEAD != nil {
 		return m.rotateEncryptionKey()
 	}
-	return errors.New("must specify one of `--current-db-version`, `--supported-db-version`, `--migrate-db-to-version`, or `--old-encryption-key`")
+	return errors.New("must specify one of `--current-db-version`, `--supported-db-version`, `--migrate-db-to-version`, `--old-encryption-key`, or `--migrate-to-latest-version`")
```

Comment:
Small nit: can you move this to the front so that it would display the flags in the order they're evaluated in? If for some reason I passed in both `--current-db-version` and `--migrate-db-to-version 6.0`, I would expect it to show the current version first and then migrate.

Reply:
Great point! changed it 👍